app/models/claims/clawback.rb

```diff
@@ -0,0 +1,11 @@
+# == Schema Information
+#
+# Table name: clawbacks
+#
+#  id         :uuid             not null, primary key
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+class Claims::Clawback < ApplicationRecord
+  has_many :clawback_claims
```

Comment:
We may need access to the claims directly for the `claims_activity` model

```suggestion
  has_many :clawback_claims
  has_many :claims, through :clawback_claims
```